index.html

```diff
@@ -10751,6 +10751,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>Indicates that the <a>element</a> is <a>perceivable</a> but disabled, so it is not editable or otherwise <a>operable</a>. See related <sref>aria-hidden</sref> and <pref>aria-readonly</pref>.</p>
 				<p>For example, irrelevant options in a radio group may be disabled. Disabled elements might not receive focus from the tab order. For some disabled elements, applications might choose not to support navigation to descendants. In addition to setting the <sref>aria-disabled</sref> attribute, authors SHOULD change the appearance (grayed out, etc.) to indicate that the item has been disabled.</p>
 				<p>The <a>state</a> of being disabled applies to the current element and all focusable descendant elements of the element on which the <sref>aria-disabled</sref> <a>attribute</a> is applied.</p>
+                <p class="note">While <sref>aria-disabled</sref> and proper scripting can successfully disable an element with role <rref>link</rref>, fully disabling a host language equivalent can be problematic. Authors are advised not to use <sref>aria-disabled</sref> on elements that cannot be fully disabled through author scripting alone.</p>
```

Comment:
This bit is a little confusing to me:
>  elements that cannot be fully disabled through author scripting alone

since most things (including links) can be made impossible to interact with through author scripting, and my gut developer reaction to the wording is "challenge accepted" 😆 ,

Would something like this be worth considering?:
>  elements that cannot be fully disabled through HTML alone

with the intent that native HTML elements that do not support the `disabled` attribute also shouldn't support `aria-disabled` (though that's also my editorial opinion 😄)

Reply:
To clarify what I mean by links can be disabled through author scripting:

In addition to `pointer-events: none` and preventing click/keydown/etc, I could work around the context menu issue by cloning the `<a>`, removing the `href`, and programmatically calling a `contextmenu` event on the cloned node. Then you'd get a comparable context menu without the "open link" options.

It's a completely silly exercise of course, and no one would do it in an actual product, but it's why I'm leery of saying "this is impossible with scripting"

Reply:
how about 
>elements that cannot be disabled through features of the host language alone.

that keeps this agnostic rather than singling out HTML links, and yes, removes the "challenge" aspect of the advisement

Reply:
Or possibly (fewer words `<g>`)
> elements that cannot be disabled using the host language.